### DISCUSSION TITLE
[Feature Request] Copy path from file properties

### DISCUSSION THREAD

**Discussion by raspin0** [linuxdeepin/developer-center#2911] [Features Request & Ideas | 特性请求 & 头脑风暴]
Hello, currently information about Location of a file in the properties isn't really useful. I wish there was an option just to click Location section to copy a path of your file.

![image](https://user-images.githubusercontent.com/39202451/138529139-b25c149f-a34e-47a7-8f58-634bf2b50cfc.png)

**Comment by meiyixiang** [comment]
This is a good suggestion and I have recorded that I will optimize this feature later

**Comment by meiyixiang** [comment]
I understand what you mean. I have fed back your suggestions to the relevant product manager
